recipe/build.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-./configure --prefix=$PREFIX
+./configure --prefix=$PREFIX PERL='/usr/bin/env perl'
```

Comment:
You correctly have `perl` on your env. Not sure what is the problem there. Maybe https://github.com/conda-forge/texinfo-feedstock/pull/5#issuecomment-260755676 will help you understand what conda is doing there.